CHANGELOG.md

```diff
@@ -7,6 +7,15 @@
   compatibility with chains started with P5 genesis data, where some genesis bakers are not set to
   restake earnings. Other chains (including mainnet and testnet) are not affected.
 - Support the new ConcordiumBFT consensus.
+- Changes to the `GetConsensusStatus` endpoint:
+  * Slot duration only returned in consensus version 1.
```

Comment:
I think it would be better to use protocol versions in the changelog rather than consensus versions. 

The chagnelog is meant for external consumption